spec/Section 4 -- Introspection.md

```diff
@@ -232,13 +235,16 @@ actually valid. These kinds are listed in the `__TypeKind` enumeration.
 
 Represents scalar types such as Int, String, and Boolean. Scalars cannot have fields.
 
-A GraphQL type designer should describe the data format and scalar coercion
-rules in the description field of any scalar.
+A GraphQL type designer should use the `@specified` directive to provide
+a specification URL describing the data format, serialization, and coercion
+rules for the scalar. It is recommended to also summarize the format and provide
+an example in the description field.
 
 Fields
 
 * `kind` must return `__TypeKind.SCALAR`.
 * `name` must return a String.
+* `specifiedBy` may return a URL or {null}.
```

Comment:
We don't have a `URL` type so it should be `String`.

Reply:
maybe `may return a String (in the form of a URL) or {null}.` ?